Physics/HelicityFormalism/HelicityDecay.hpp

```diff
@@ -49,6 +51,9 @@ class HelicityDecay : public NamedAmplitude {
   std::shared_ptr<ComPWA::Physics::Dynamics::AbstractDynamicalFunction>
       DynamicFunction;
 
+  std::shared_ptr<ComPWA::Physics::Dynamics::AbstractDynamicalFunction>
```

Comment:
I would remove this (actually all of you changes in the HelicityDecay class), and instead go for the decorator pattern.